Makefile

```diff
@@ -549,6 +549,8 @@ else
 endif
 	$(MAKE) -C tools installopt
 
+ARCHIVES = ocamlcommon ocamlbytecomp ocamloptcomp
```

Comment:
Not strictly related to this PR, but shouldn't `ocamlmiddleend.$(A)` and `ocamltoplevel.$(A)` be included?

Reply:
I'm afraid that's beyond my knowledge level! :-)